frontend/src/redux/actions/actions.ts

```diff
@@ -23,16 +45,36 @@ export const getComponentsRejected = (error) => ({
   },
 });
 
-export const getComponents = () => {
+export const getComponents = (installed: boolean = false) => {
```

Comment:
added an eslint "warn" on this instead of an error "@typescript-eslint/no-inferrable-types" feel free to adjust